<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberItemWidget</class>
 <widget class="QWidget" name="MemberItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>70</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MemberItemWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MemberItemWidget{background-color: rgb(255, 255, 255);};</string>
  </property>
  <widget class="QWidget" name="FilletWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>730</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>730</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>730</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#FilletWidget{background:rgb(252,252,252);border-radius:4px;border:1px solid rgb(235,235,235);}
QWidget#FilletWidget:hover,pressed{background:rgb(246,251,255);border-radius:4px;border:1px solid rgb(187,223,255);};</string>
   </property>
   <widget class="QLabel" name="labelJob">
    <property name="geometry">
     <rect>
      <x>503</x>
      <y>22</y>
      <width>180</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
background-color: rgba(255, 255, 255, 0);
border:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelMobile">
    <property name="geometry">
     <rect>
      <x>324</x>
      <y>22</y>
      <width>178</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>178</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
background-color: rgba(255, 255, 255, 0);
border:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelFirstName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="operButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>10</width>
      <height>18</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>18</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>10</width>
      <height>18</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/EntCenterWidget/image/oper.png);}
QPushButton::menu-indicator{image:none;}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelUserName">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>22</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
background-color: rgba(255, 255, 255, 0);
border:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTrueName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>22</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
background-color: rgba(255, 255, 255, 0);
border:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EntCenterWidget.qrc"/>
 </resources>
 <connections/>
</ui>
